docs/Data Model.md

```diff
@@ -8,11 +8,11 @@ In order to ensure consistent behaviour in a facility employing this identity, s
 
 ![Extended IS-04 Model Diagram](../docs/images/is-11-data-model.png)
 
-The IS-04 hierarchy is preserved. Additionally, Senders managed with IS-11 SHOULD be associated with 1 or more Input resources and Receivers managed with IS-11 SHOULD be associated with 1 or more Output resources.
+The IS-04 hierarchy is preserved. Additionally, there are Input and Output resources introduced. Senders managed with IS-11 are associated with 0 or more Input resources and Receivers managed with IS-11 are associated with 0 or more Output resources.
```

Comment:
"are" of "MAY be" associated ... ?

Reply:
MAY expresses an OPTIONAL item. If it said 1 or more, I think that would clearly be inappropriate since making it optional would mean some implementations could have 0 and that would be permitted. So here, where we are saying 0 or more, I think it is probably better not to use conformance language at all, and use "are" or "can be"?